app/views/shared/_service_comparison.html.erb

```diff
@@ -0,0 +1,360 @@
+            </div>
+            <div class="service diy">
+              <%= image_tag "icons/check-outline.svg", alt: "yes" %>
+            </div>
+            <div class="service express">
+            </div>
+          </div>
+        </div>
+        <div class="service-set">
+          <div class="service-set__header">
+            <div class="service-set__title service-set__title">
+              <%= t("views.shared.service_comparison.additional_benefits") %>
+            </div>
+          </div>
+          <div class="service-set__services service-set--mobile-excluded">
+            <div class="service full-service">
+            </div>
+            <div class="service diy">
+
+            </div>
+            <div class="service express">
+            </div>
+          </div>
+        </div>
+        <div class="service-set">
+          <div class="service-set__header">
+            <div class="service-set__title service-set__title--normal">
+              <%= t("views.shared.service_comparison.chat_support") %>
+            </div>
+          </div>
+          <div class="service-set__services">
+            <div class="service full-service service--mobile-flex-start">
+              <%= image_tag "icons/check-outline.svg", alt: "yes" %>
+            </div>
+            <div class="service diy service--mobile-flex-start service--fixie-parent">
+              <%= image_tag "icons/check-outline.svg", alt: "yes" %>
+              <div class="service-set__subtitle service-set__subtitle--fixie">
+                <%= t("views.shared.service_comparison.support_via_gyr") %>
+              </div>
+            </div>
+            <div class="service express service--mobile-flex-start">
+              <%= image_tag "icons/check-outline.svg", alt: "yes" %>
+            </div>
+          </div>
+        </div>
+        <div class="service-set">
+          <div class="service-set__header">
+            <div class="service-set__title service-set__title--normal">
+              <%= t("views.shared.service_comparison.itin_assistance") %>
+            </div>
+          </div>
+          <div class="service-set__services">
+            <div class="service full-service">
+              <%= image_tag "icons/check-outline.svg", alt: "yes" %>
+            </div>
+            <div class="service diy">
+            </div>
+            <div class="service express">
+            </div>
+          </div>
+        </div>
+        <div class="service-set">
+          <div class="service-set__header">
+            <div class="service-set__title service-set__title--normal">
+              <%= t("views.shared.service_comparison.vita") %>
+            </div>
+          </div>
+          <div class="service-set__services">
+            <div class="service full-service">
+              <%= image_tag "icons/check-outline.svg", alt: "yes" %>
+            </div>
+            <div class="service diy">
+
+            </div>
+            <div class="service express">
```

Comment:
These might be tabular data, which might be a good use of tables, but it's really fine the way it is, too. When reading the HTML, it took me a while to realize that these divs were really table cells (and empty table cells at that).